File: DTOs/Sources/PollServiceDTOs/CreatePollDTO.swift
import Foundation

public struct CreatePollDTO: Codable {
    public var question: String
    public var description: String?
    public var closedAt: Date
    public var anonymous: Bool
    public var multiSelect: Bool
    public var options: [GetPollVotingOptionDTO]
    
    public init(question: String, description: String? = nil, closedAt: Date, anonymous: Bool, multiSelect: Bool, options: [GetPollVotingOptionDTO]) {
        self.question = question
        self.description = description
        self.closedAt = closedAt
        self.anonymous = anonymous
        self.multiSelect = multiSelect
        self.options = options
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/PollServiceDTOs/CreatePollDTO.kt
package net.ipv64.kivop.dtos.PollServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class CreatePollDTO (
    var question : String,
    var description : String?,
    var closedAt : LocalDateTime,
    var anonymous : Boolean,
    var multiSelect : Boolean,
    var options : List<GetPollVotingOptionDTO>,
)
